Accel.cs

```diff
@@ -9,16 +9,39 @@ public class Accel : IOModule {
         bool doX = false;
         bool doY = false;
         bool doZ = false;
-        public Accel(string xyz) {
-            xyz = xyz.ToLower();
-            switch (xyz) {
-                case "x":
+        //public Accel(string xyz) {
+        //    xyz = xyz.ToLower();
+        //    switch (xyz) {
+        //        case "x":
+        //            this.doX = true;
+        //            break;
+        //        case "y":
+        //            this.doY = true;
+        //            break;
+        //        case "z":
+        //            this.doZ = true;
+        //            break;
+
+        //    }
+
+        //    if (this.doX == false && this.doY == false && this.doZ == false)
+        //        throw new ArgumentException("Argument must be X or Y or Z");
+
+        //    if (Controller.IsPulse) {
+        //        var i2ccon = I2cController.FromName(SC13048.I2cBus.I2c2);
+        //        this.accel = new MC3216Controller(i2ccon);
+        //    }
+        //}
+
+        public Accel(double xyz) {            
```

Comment:
After playing with this, how do I get both the X and Y-axis? For example, I wanted to create a game that you had to balance a ball on the screen in both the X and Y direction, but with this interface, I can only do one or the other but not both at the same time.